src/storage/v2/storage.hpp

```diff
@@ -224,6 +227,26 @@ class Storage {
 
     virtual bool LabelPropertyIndexExists(LabelId label, PropertyId property) const = 0;
 
+    bool TextIndexExists(std::string index_name) const {
```

Comment:
How we support all of this for disk storage? What is the logic if data is in RocksDB and then where else also?

Reply:
The text indices are in Tantivy regardless whether Memgraph uses in-memory or on-disk storage. Because of this, `TextIndexExists` (and related methods below) are implemented in the superclass.